web/src/lib/components/shared-components/tree/tree-item-thumbnails.svelte

```diff
@@ -12,7 +12,7 @@
 
 {#if items.length > 0}
   <div
-    class="w-full grid grid-cols-2 sm:grid-cols-4 lg:grid-cols-6 2xl:grid-cols-8 gap-2 bg-gray-50 dark:bg-immich-dark-gray/50 rounded-2xl border border-gray-100 dark:border-gray-900 max-h-[500px] overflow-auto immich-scrollbar"
+    class="w-full grid grid-cols-2 sm:grid-cols-4 lg:grid-cols-6 2xl:grid-cols-8 gap-2 bg-gray-50 dark:bg-immich-dark-gray/50 rounded-2xl border border-gray-100 dark:border-gray-900 overflow-auto immich-scrollbar"
```

Comment:
The breadcrumb navigation bar scrolls out of view due to this change. It would be better if it remained visible at the top at all times.

Since this div no longer overflows, these classes can be removed:
```suggestion
    class="w-full grid grid-cols-2 sm:grid-cols-4 lg:grid-cols-6 2xl:grid-cols-8 gap-2 bg-gray-50 dark:bg-immich-dark-gray/50 rounded-2xl border border-gray-100 dark:border-gray-900"

```

Reply:
yes,It is better to fix it at the top.